nwbwidgets/test/test_processing_module_widgets.py

```diff
@@ -0,0 +1,84 @@
+
+import numpy as np
+from pynwb import NWBFile
+from nwbwidgets.base import processing_module
+from datetime import datetime
+from dateutil.tz import tzlocal
+from pynwb.behavior import Position
+from pynwb.behavior import SpatialSeries
+from pynwb import ProcessingModule
+
+
+def test_processing_module():
+    
+    # Create a NWB file
+    start_time = datetime(2020, 1, 29, 11, tzinfo=tzlocal())
+    nwbfile = NWBFile(session_description = 'Test Session',  

```

Comment:
I use PEP8 as a style guide, which advises ["Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter."](https://www.python.org/dev/peps/pep-0008/#other-recommendations)